applications/HDF5Application/python_scripts/core/xdmf.py

```diff
@@ -328,7 +328,10 @@ def attribute_type(self):
         if len(self.data.dimensions) == 1:
             return "Scalar"
         elif len(self.data.dimensions) == 2:
-            return "Vector"
+            if (self.data.dimensions[1] == 3):
```

Comment:
It is because when the size is 3, paraview shows them as _X, _Y, _Z. If the size is more than three, then paraview cannot read "Vector", it needs to be "Matrix" to visualize all the components. Then it is showed as _1, _2, _3, _4, ...